lib/govuk_publishing_components/presenters/machine_readable/is_part_of_schema.rb

```diff
@@ -0,0 +1,20 @@
+module GovukPublishingComponents
+  module Presenters
+    class IsPartOfSchema
+      attr_reader :is_part_of_url
```

Comment:
This is quite pernickety on my part but this attr_reader isn't necessary (nothing else uses it) and nothing else really needs to use it so I'd remove it